<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1258</width>
    <height>851</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>База данных договоров</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="user">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>966</width>
        <height>24</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="loginButton">
      <property name="toolTip">
       <string>Войти в систему</string>
      </property>
      <property name="text">
       <string>Авторизация</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="changePasswordButton">
      <property name="toolTip">
       <string>Открыть окно смены пароля</string>
      </property>
      <property name="text">
       <string>Сменить пароль</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="4">
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::West</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="applicationTab">
       <attribute name="title">
        <string>Приложения</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,0,1,0,1,0,1,0,1,0,1">
        <item row="0" column="0">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Сектор:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" rowspan="2">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Договор:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="2">
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Тема:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="5" rowspan="2">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="6">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Тип:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="7" rowspan="2">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="8">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Организация:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="9" rowspan="2">
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="10" rowspan="2">
         <widget class="QGroupBox" name="dateGroupBox">
          <property name="title">
           <string>Дата:</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="1,3" columnminimumwidth="0,0">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>С:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>По:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="MyDateEdit" name="sinceDate" native="true"/>
           </item>
           <item row="1" column="1">
            <widget class="MyDateEdit" name="toDate" native="true"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="MyComboBox" name="secCombo"/>
        </item>
        <item row="1" column="4">
         <widget class="MyComboBox" name="themeCombo"/>
        </item>
        <item row="1" column="6">
         <widget class="MyComboBox" name="typeCombo"/>
        </item>
        <item row="1" column="8">
         <widget class="MyComboBox" name="orgCombo"/>
        </item>
        <item row="2" column="0" colspan="11">
         <widget class="Line" name="line_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="0" colspan="11">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="addApplicationButton">
            <property name="toolTip">
             <string>Добавить новое приложение</string>
            </property>
            <property name="text">
             <string>Добавить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="deleteApplicationButton">
            <property name="toolTip">
             <string>Удалить выбранные приложения</string>
            </property>
            <property name="text">
             <string>Удалить</string>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="QPushButton" name="filterButton">
            <property name="toolTip">
             <string>Открыть фильтр выбора данных из БД</string>
            </property>
            <property name="text">
             <string>Отобразить фильтр</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QPushButton" name="resetButton">
            <property name="toolTip">
             <string>Сбросить настройки фильтра</string>
            </property>
            <property name="text">
             <string>Очистить</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0" colspan="11">
         <widget class="QTableView" name="applicationTable"/>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="dealEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dealTab">
       <attribute name="title">
        <string>Договора</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_10">
        <item row="1" column="0" colspan="3">
         <widget class="QTableView" name="dealTable"/>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="deleteDealButton">
          <property name="toolTip">
           <string>Удалить выбранные договоры</string>
          </property>
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addDealButton">
          <property name="toolTip">
           <string>Добавить договор</string>
          </property>
          <property name="text">
           <string>Добавить</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">mycombobox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyDateEdit</class>
   <extends>QWidget</extends>
   <header location="global">mydateedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
